core/templates/dev/head/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.html

```diff
@@ -44,6 +44,7 @@
           <li class="nav-item"> <a class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-get-started-link " href="/get_started" translate="I18N_TOPNAV_GET_STARTED"></a></li>
           <li class="nav-item"> <a class="dropdown-item nav-link oppia-navbar-tab-content protractor-test-playbook-link " href="/teach#playbook" translate="I18N_TOPNAV_PARTICIPATION_PLAYBOOK"></a></li>
           <li class="nav-item"> <a class="dropdown-item nav-link oppia-navbar-tab-content" href="https://medium.com/oppia-org" target="_blank" rel="noopener" translate="I18N_TOPNAV_BLOG">Blog</a></li>
+          <li class="nav-item"> <a class="dropdown-item nav-link oppia-navbar-tab-content"  href="/contact" target="_blank" rel="noopener" translate="I18N_TOPNAV_CONTACT"></a></li>
```

Comment:
Why `target="_blank" rel="noopener"`?

Reply:
target="_blank " just to open the link in new tab

Reply:
But why to open it in new tab, we use new tab for pages on different domain.

Reply:
I have opened it in a new tab just because all the other links like Blog, About Oppia, etc  are opened in that way, if you don't want it I will change it

Reply:
Yeah no problem, but blog is on other domain. About Oppia is opened in the same tab. Please change it to the same tab for the contacts link.